### PR TITLE
Merge yo office custom functions template

### PR DESCRIPTION
**Pull request by TCourtneyOwen**
Currently we have two distinct custom functions templates, one that lives in the Excel-Custom-Functions repo and another that lives under the generator-office repo. This PR is the first step in having a single custom functions template by replacing the existing project template for the Excel-Custom-Functions repo  with the Yo Office instance.  Updating the Excel-Custom-Functions repo  so it matches Yo Office also gets us one steps closer to having the existing Yo Office project templates each living in their own separate repo, so that when we need to make a fix or change to a particular template, we don't have to publish a whole new version of Yo Office.  One final and important benefit of merging the Yo Office version is that we now get sideloading (npm run sideload) and the ability to sideload in XL Online with the addition of a web.config that enables CORS

### CODE REVIEW

**Comment by mscharlock** [comment]
Everything you've said makes sense to me - we should have some consistency and the npm run sideload is a ton easier than the other thing. @Reezaali or @keyur32 does getting rid of lines 36-60 in the Readme.md make sense? I don't know if I have enough context to make that call.

File: README.md [hunk 1]
```diff
@@ -1,63 +1,40 @@
...
+3. Run `npm run sideload` to launch Excel and load the custom functions add-in. Additonal information on sideloading can be found at <https://aka.ms/sideload-addins>.
+4. After Excel launches, you will need to register the custom-functions add-in to work around a bug:
+    a. On the upper-left-hand side of Excel, there is a small hexagon icon with a dropdown arrow. The icon is to right of the Save icon.
+    b. Click on this dropdown arrow and then click on the Custom Functions Sample add-in to register it.
+5. Test a custom function by entering `=CONTOSO.ADD42(num)` in a cell.
+6. Try the other functions in the sample: `=CONTOSO.ADD42ASYNC(num, num)`, `CONTOSO.ISPRIME(num)`, `CONTOSO.NTHPRIME(num)`, `CONTOSO.GETDAY()`, `CONTOSO.INCREMENTVALUE(increment)`, and `CONTOSO.SECONDHIGHEST(range)`.
+7. If you make changes to the sample add-in, copy the updated files to your website, and then close and reopen Excel. If your functions are not available in Excel, re-insert the add-in using **Insert** > **My Add-ins**.
+8. Follow @OfficeDev on Twitter for updates and send feedback to <excelcustomfunctions@microsoft.com>.
 
-## Making changes
```
Comment (mscharlock) [inline README.md]: Do we not need 36-60 any more? Just curious! They seem useful to me but maybe I'm lacking the context for why you removed them.

Comment (mscharlock) [inline README.md]: Do we not need 36-60 any more? I might be lacking context about why these got removed.

Comment (Reezaali) [reply README.md]: @mscharlock can you verify with Courtney and Sylvia please? Thanks!

Comment (TCourtneyOwen) [reply README.md]: I think we can add back the JSON intellisense and debugging steps  I will add them back into the ReadMe

**Comment by TCourtneyOwen** [comment]
I would like to get this pushed out to main today if possible.  Are there any other concerning issues with this PR?

**Comment by TCourtneyOwen** [comment]
Thanks! Merging to master now

**Pull request closed without merging**